firedrake/mesh.py

```diff
@@ -902,10 +906,11 @@ def make_offset(self, entity_dofs, ndofs):
             entity_offset[b] += len(entities[0])
 
         dof_offset = np.zeros(ndofs, dtype=IntType)
-        for (b, v), entities in entity_dofs.items():
-            for dof_indices in entities.values():
-                for i in dof_indices:
-                    dof_offset[i] = entity_offset[b]
```

Comment:
I wonder if it works to `return None` in the real case.  That is more "correct"

Reply:
Doesn't seem to work out of the box.

Reply:
Might work in TJ's new reworking of the pyop2 codegen (I guess this is where things fail?).  Let's keep it in the back of our minds.

Reply:
OK